proto/v1alpha1/resource.proto

```diff
@@ -50,13 +50,25 @@ message LabelTerm {
         EXISTS = 0;
         // Label value is equal.
         EQUAL = 1;
-        // Label doesn't exist.
```

Comment:
I don't think this is the way to deprecate, maybe there's a proper way? :thinking:

Reply:
we can do `NOT_EXISTS = 2 [deprecated=true];`, will update.